Source/Core/Common/CommonPaths.cpp

```diff
@@ -0,0 +1,536 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Common/CommonPaths.h"
+#include "Common/FileUtil.h"
+#include "Common/Logging/Log.h"
+
+#ifdef _WIN32
+#include <windows.h>
+#include "Common/StringUtil.h"
+#else
+#include <limits.h>
+#include <unistd.h>
+#endif
+
+#ifdef __APPLE__
+#include <CoreFoundation/CFBundle.h>
+#include <CoreFoundation/CFString.h>
+#include <CoreFoundation/CFURL.h>
+#include <sys/param.h>
+#endif
+
+// Shared data dirs (Sys and shared User for Linux)
+constexpr auto SYSDATA_DIR =
+#if defined(_WIN32) || defined(LINUX_LOCAL_DEV)
+    "Sys"
+#elif defined __APPLE__
```

Comment:
Is there any reason for being inconsistent, with `_WIN32` and `LINUX_LOCAL_DEV` in braces but `__APPLE__` etc without?